spec/features/houses/show_house_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe "show house details", type: :feature do
+  let!(:house1) do
+    FactoryBot.create(
+      :house,
+      number: "205B",
+      address: "B apartament block",
```

Comment:
you don't need to specify `address`, `route` and `status` unless you are testing them... ATM you are testing only the `number`